### DISCUSSION TITLE
accuracy of synced external axis movement

### DISCUSSION THREAD

**Discussion by lin-ycv** [visose/Robots#298] [Q&A] [answered]
I'm wondering if moving an external axis rotary and/or linear rail while milling would impact the accuracy in any meaningful way, if my tolerances requirements are only around 1mm.

Talked to a few SI in my region, they all recommend to not move external axis while milling, they claim it's slower than milling area by area, but didn't mention anything about accuracy.

**Comment by visose** [comment, marked as answer]
This would take long to answer but in practice probably the less moving parts the better. All mechanisms have some manufacturing inaccuracies which add up when all are moving at once, but I doubt this would be the accuracy bottleneck. It will likely be in things you have installed yourself, like the end effector and the way you mounted the work piece on the positioner. Even if you calibrate it very accurately things might move, bend. Best is to test and measure, to see where the actual bottleneck is.
